RELEASE_NOTES.md

```diff
@@ -7,8 +7,8 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# azure-chef-extension 1210.12.100.1000 release notes:
-In this release we have added functionality not to uninstall chef-client even if the extension gets uninstalled.
+# azure-chef-extension 1210.12.101.1000 for Windows and 1210.12.101.1000 for Ubuntu release notes:
+In this release we have added extension support for Debian platform, added functionality to determine whether to uninstall chef-client or not using flag uninstall_chef_client and installing chef from Omnitruck for Ubuntu CentOS and Debian
 
 See the [CHANGELOG](https://github.com/chef-partners/azure-chef-extension/blob/master/CHANGELOG.md) for a list of all changes in this release, and review.
 

```

Comment:
We need to add one more section for `Features added in azure-chef-extension 1210.12.101.1000 and 1210.12.101.1001`.
Please refer this https://github.com/chef-partners/azure-chef-extension/commit/20f4baa3c4cfcdcf6d166ed7b5a2f9b561730873#diff-f54e6ba45d43eee7cdc25838b79b012cL17